sentinel-transport/sentinel-transport-netty-http/src/test/java/com/alibaba/csp/sentinel/transport/command/netty/HttpServerHandlerTest.java

```diff
@@ -152,60 +164,125 @@ public void testGetRuleCommandFlowSomeRules() {
         String httpRequestStr = "GET /getRules?type=flow HTTP/1.1" + CRLF
                               + "Host: localhost:8719" + CRLF
                               + CRLF;
-        String body = "";
 
-        processSuccess(httpRequestStr, body);
+        // body json
+        /*
+        String expectedBody = "[{\"clusterMode\":false,\"controlBehavior\":0,\"count\":20.0"
+                + ",\"grade\":1,\"limitApp\":\"default\",\"maxQueueingTimeMs\":500"
+                + ",\"resource\":\"key\",\"strategy\":0,\"warmUpPeriodSec\":10}]";
+        */
+        String expectedBody = JSON.toJSONString(rules);
+
+        processSuccess(httpRequestStr, expectedBody);
+    }
+
+    /**
+     * {@link com.alibaba.csp.sentinel.cluster.server.command.handler.FetchClusterFlowRulesCommandHandler}
+     *
+     * Note:
+     * To test the command whose mapping path and command name contain /
+     *
+     * The mapping path is /cluster/server/flowRules
+     * The command name is cluster/server/flowRules
+     */
+    @Test
+    public void testFetchClusterFlowRulesCommandEmptyRule() {
+        String httpRequestStr = "GET /cluster/server/flowRules HTTP/1.1" + CRLF
+                + "Host: localhost:8719" + CRLF
+                + CRLF;
+        String expectedBody = "[]";
+
+        processSuccess(httpRequestStr, expectedBody);
+    }
+
+    @Test
+    public void testFetchClusterFlowRulesCommandSomeFlowRules() {
```

Comment:
Introducing `sentinel-cluster-server-default` here causes the test case surrounded by the logic of loading cluster rules, which might be too complicated. How about we add a test handler (with multiple `/` in its name) and just test for it?

Reply:
Thanks, really a good idea!
The `sentinel-cluster-server-default` dependency has been removed.
A `MultipleSlashNameCommandTestHandler` class is added, with TestHandler suffix means that it's a test handler. Its command name has multiple /, the resources folder and META-INF/services are added in test folder, to load the Handler by SPI.
Now the test case method is `testMultipleSlashNameCommand`.